### ISSUE TITLE
Cannot rename existing protocol

### ISSUE THREAD

**Issue by Ducasse** [guillep/pharo-core#15804]
In latest P12 I cannot rename protocol.

- Select one
- select rename and pick up a new one, press ok
- Nothing changes

**Comment by Ducasse** [comment]
https://github.com/pharo-project/pharo/assets/432549/a3e2d862-5451-491a-a5cf-cf770e0daabb

**Comment by Ducasse** [comment]
In addition the remove protocol does not work anymore. 
I do not get why. I checked the code of ClyMethodGroup and ClyRemoveMethodGroupCommand 
in P11 and I could not find in P11 how the protocol was removed. Actually the contained methods are removed but not the protocol. May be there is a side effect when removing the last method that removes the protocol too. 

BTW I should say that I do not see the value of changing

```
removeWithMethods
	self methods do: [ :each | each removeFromSystem ]
```

into 

```
removeWithMethods
	self methods do: #removeFromSystem
```

What is the gain?
